ceph.go

```diff
@@ -116,6 +116,15 @@ type pgQueryOut struct {
 	} `json:"peer_info"`
 }
 
+// mappingsAsMap returns the To/From OSD mapping pairs of an upmap item as a map
+func (pui *pgUpmapItem) mappingsAsMap() map[int]int {
```

Comment:
It would be nice to have the "sense" of the map in the name of the function so that it's clear what the key is. `mappingsAsToFromMap` is the first thing that came to mind, but it's a bit awkward...